src/plugins/intel_cpu/src/nodes/random_uniform.cpp

```diff
@@ -18,10 +18,6 @@ bool RandomUniform::isSupportedOperation(const std::shared_ptr<const ov::Node>&
             errorMessage = "Only RandomUniform operation from the opset8 is supported by the CPU plugin.";
             return false;
         }
-        if (as_type_ptr<const op::v8::RandomUniform>(op)->get_alignment() != op::PhiloxAlignment::TENSORFLOW) {
```

Comment:
This fn serves for external validation as well and need to check attributes. In ctr you can check the correctness of the operation.

Reply:
This check is no longer needed as the alignment argument can take both PHILOX and MERSENNE_TWISTER, and the correctness is verified in the ctr as you said :)